sources/packages/backend/apps/api/src/route-controllers/cas-invoice-batch/cas-invoice-batch.aest.controller.ts

```diff
@@ -28,10 +43,13 @@ import {
 } from "@sims/utilities";
 import { Response } from "express";
 import { streamFile } from "../utils";
-import { CAS_INVOICE_BATCH_NOT_FOUND } from "../../constants";
+import {
+  CAS_INVOICE_BATCH_NOT_FOUND,
+  CAS_INVOICE_BATCH_NOT_PENDING,
+} from "../../constants";
 
 @AllowAuthorizedParty(AuthorizedParties.aest)
-@Roles(Role.AESTCASInvoicing)
+@Roles(Role.AESTCASInvoicing, Role.AESTCASInvoiceBatchExpense)
```

Comment:
When we have multiple roles, we associate them individually to each endpoint.
Since this controller now has multiple roles to different endpoints, please remove the `@Roles` at controller level and have the existing `Role.AESTCASInvoicing` associated with the previously existing endpoints.